test/Microsoft.IdentityModel.TestUtils/Default.cs

```diff
@@ -1239,6 +1259,11 @@ public static TokenValidationParameters JWECompressionTokenValidationParameters
             }
         }
 #endif
+        public static string Tid
```

Comment:
This does not quite make sense.
The Default class is for the value of claims, not the type of the claim.

In AadPayloadString just use "tid' as the claim name and return "tenantId' here.

Reply:
removed the Tid